src/components/flyout/index.ts

```diff
@@ -0,0 +1,7 @@
+export { EuiFlyout, EuiFlyoutProps, SIZES } from './flyout';
```

Comment:
```suggestion
export { EuiFlyout, EuiFlyoutProps, SIZES as FLYOUT_SIZES } from './flyout';
```

Resulted in a TS build error:

> 08:15:29 Generating typescript definitions file
> 08:16:02 eui.d.ts(3288,39): error TS2300: Duplicate identifier 'SIZES'.
> 08:16:02 eui.d.ts(5755,38): error TS2300: Duplicate identifier 'SIZES'.

Open to other naming suggestions, also, because this isn't the only place we'll need adjust export names //cc @chandlerprall